### PR TITLE
Add test for initial setup view controllers.

### PR DESCRIPTION
**Pull request by Ivan-Skoryk**
## Summary
Add test for initial setup view controllers.

 #### Checklist:
 - [x] I have added a brief description of the problem in the PR body.
 - [x] I have explained how I solved the problem in the PR body.
 - [x] I have covered possible edge cases and side effects.
 - [x] I have have attached screenshots to the PR of any UI change.
 - [x] I have checked that my changes work on iOS and macOS targets.
 - [x] I have checked that my changes work correctly on devices with and without a Home button.
 - [x] I have added unit tests for the new logic.
 - [x] I have achieved maximum possible code coverage for the new code.
 - [x] I have checked that my UI works correctly on both Light and Dark themes.

### CODE REVIEW

File: xDripTests/Scenes/InitialSetup/ViewControllers/Generic/InitialSetupTransmitterTypeViewController/InitialSetupTransmitterTypeViewControllerTests.swift [hunk 1]
```diff
@@ -7,6 +7,45 @@
...
+        }
+        
+        func doCompleteCustomDeviceStep(request: InitialSetup.CompleteCustomDeviceStep.Request) { }
+    }
+    
+    func testOnDexcomG6Selected() {
+        let spy = InitialSetupBusinessLogicSpy()
+        sut.interactor = spy
+        
+        guard let button = sut.view.subviews.first(where: { $0.accessibilityIdentifier == "dexcomG6Button" }) as? UIButton else {
```
Comment (Artem-Kalmykov) [inline xDripTests/Scenes/InitialSetup/ViewControllers/Generic/InitialSetupTransmitterTypeViewController/InitialSetupTransmitterTypeViewControllerTests.swift:39]: Create a recursive method to search for a view in all subviews.

File: xDrip/Scenes/InitialSetup/ViewControllers/G6/InitialSetupG6ConnectViewController/InitialSetupG6ConnectViewController.swift [hunk 2]
```diff
@@ -28,16 +42,16 @@ final class InitialSetupG6ConnectViewController: InitialSetupAbstractStepViewCon
         interactor?.doCompleteCustomDeviceStep(request: request)
     }
     
-    private let worker = InitialSetupDexcomG6ConnectionWorker()
+    private var worker: InitialSetupDexcomG6ConnectionWorkerProtocol?
```
Comment (Artem-Kalmykov) [inline xDrip/Scenes/InitialSetup/ViewControllers/G6/InitialSetupG6ConnectViewController/InitialSetupG6ConnectViewController.swift:45]: Why `var` and optional? Make it `private let worker: InitialSetupDexcomG6ConnectionWorkerProtocol` and modify init to:
```swift
required init(connectionWorker: InitialSetupDexcomG6ConnectionWorkerProtocol) {
         worker = connectionWorker
         super.init()
     }
```

File: xDripTests/Scenes/InitialSetup/ViewControllers/G6/InitialSetupG6WarmUpViewController/InitialSetupG6WarmUpViewControllerTests.swift [hunk 3]
```diff
@@ -7,6 +7,43 @@
...
+        func doSaveSettings(request: InitialSetup.SaveSettings.Request) { }
+        func doSelectDeviceType(request: InitialSetup.SelectDevice.Request) { }
+        
+        func doCompleteCustomDeviceStep(request: InitialSetup.CompleteCustomDeviceStep.Request) {
+            calledCompleteSetup = true
+            moreStepsExpected = request.moreStepsExpected
+        }
+    }
+    
+    func test() {
```
Comment (Artem-Kalmykov) [inline xDripTests/Scenes/InitialSetup/ViewControllers/G6/InitialSetupG6WarmUpViewController/InitialSetupG6WarmUpViewControllerTests.swift:34]: provide a better name other than `test`

File: xDripTests/Scenes/InitialSetup/ViewControllers/Generic/InitialSetupDeviceModeViewController/InitialSetupDeviceModeViewControllerTests.swift [hunk 4]
```diff
@@ -7,6 +7,54 @@
...
+            deviceMode = request.deviceMode
+        }
+        
+        func doSelectInjectionType(request: InitialSetup.SelectInjectionType.Request) { }
+        func doSaveSettings(request: InitialSetup.SaveSettings.Request) { }
+        func doSelectDeviceType(request: InitialSetup.SelectDevice.Request) { }
+        func doCompleteCustomDeviceStep(request: InitialSetup.CompleteCustomDeviceStep.Request) { }
+    }
+    
+    func test() {
```
Comment (Artem-Kalmykov) [inline xDripTests/Scenes/InitialSetup/ViewControllers/Generic/InitialSetupDeviceModeViewController/InitialSetupDeviceModeViewControllerTests.swift:35]: The same here.

File: xDripTests/Scenes/InitialSetup/ViewControllers/Generic/InitialSetupInjectionTypeViewController/InitialSetupInjectionTypeViewControllerTests.swift [hunk 5]
```diff
@@ -7,6 +7,54 @@
...
+            calledSelectInjectionType = true
+            injectionType = request.injectionType
+        }
+        
+        func doSaveSettings(request: InitialSetup.SaveSettings.Request) { }
+        func doSelectDeviceType(request: InitialSetup.SelectDevice.Request) { }
+        func doCompleteCustomDeviceStep(request: InitialSetup.CompleteCustomDeviceStep.Request) { }
+    }
+    
+    func test() {
```
Comment (Artem-Kalmykov) [inline xDripTests/Scenes/InitialSetup/ViewControllers/Generic/InitialSetupInjectionTypeViewController/InitialSetupInjectionTypeViewControllerTests.swift:35]: And here

**Comment by Ivan-Skoryk** [comment]
@Artem-Kalmykov check pls

File: xDrip/Extensions/UView+Tests.swift [hunk 6]
```diff
@@ -0,0 +1,25 @@
+//
+//  UView+Tests.swift
```
Comment (Artem-Kalmykov) [inline xDrip/Extensions/UView+Tests.swift:2]: This file is used by unit tests target only, so move it to `xDripTests` instead of `xDrip`.

**Pull request merged**